dubbo-demo/dubbo-demo-xml/dubbo-demo-xml-provider/src/main/resources/spring/dubbo-provider.xml

```diff
@@ -29,7 +29,7 @@
     <dubbo:registry id="registry1" address="zookeeper://127.0.0.1:2181?registry-type=service"/>
 
     <dubbo:protocol name="dubbo" port="-1"/>
-    <dubbo:protocol name="rest" port="-1"/>
+    <dubbo:protocol name="tri" port="-1"/>
```

Comment:
Why modify it？ Please only modify the content related to this PR.

Reply:
> Why modify it？ Please only modify the content related to this PR.

The implementation of the REST protocol has been moved from the dubbo project to the dubbo-spi-extension project. Here is the test code, and the REST protocol loading fails when executed.

Reply:
> Why modify it？ Please only modify the content related to this PR.

Or do I need to add the corresponding dependency packages of the rest protocol? Please confirm.

Reply:
Commited rollback and submit a new issues to fix that.
https://github.com/apache/dubbo/issues/14340